samples/jobs.py

```diff
@@ -230,11 +229,11 @@ def on_update_job_execution_rejected(rejected):
     host_resolver = io.DefaultHostResolver(event_loop_group)
     client_bootstrap = io.ClientBootstrap(event_loop_group, host_resolver)
 
-    if args.use_websocket == True:
-        proxy_options = None
-        if (args.proxy_host):
-            proxy_options = http.HttpProxyOptions(host_name=args.proxy_host, port=args.proxy_port)
+    proxy_options = None
+    if (args.proxy_host):
```

Comment:
Should we check the `proxy_port` is non-zero like we do in all the other languages?

Reply:
Oh, nevermind, it's defaulted.